biojava-core/src/main/java/org/biojava/nbio/core/sequence/features/DatabaseReferenceInterface.java

```diff
@@ -30,7 +30,9 @@
  * has a list of cross reference id(s)
  * @author Scooter Willis <willishf at gmail dot com>
  */
+@Deprecated  //use feature interface
```

Comment:
Can you give the tip for what to use without comments? Like that it won't show in javadoc, i.e. something like 
`@Deprecated Use Feature interface instead`